### ISSUE TITLE
Charging 18650

### ISSUE THREAD

**Issue by kaelsaes** [LilyGO/TTGO-T-Beam#39]
Hi,

sorry for that newbie question but is it possible to charge the 18650 via the micro USB port?

Thanks

**Comment by sfambach** [comment]
Yes it is possible. But I am not sure if the setup of the loading controller needs some tuning to get better loading results.
